flexmeasures/ui/utils/color_defaults.py

```diff
@@ -0,0 +1,71 @@
+from flexmeasures.data.models.user import Account
+from typing import Optional
+
+
+def get_color_settings(account: Optional[Account]) -> dict:
+    """
+    This function returns the primary and secondary color settings for the UI.
+
+    It Also provides variations of the primary and secondary colors, such as border color, hover color, and transparent color.
+    """
+
+    primary_color: str = "#1a3443"
+    secondary_color: str = "f1a122"
+    if account:
+        primary_color = str(
+            account.primary_color
+            or (
+                account.consultancy_account
+                and account.consultancy_account.primary_color
+            )
+            or "#1a3443"
```

Comment:
I guess this could say `or primary_color` right? (then we don't repeat the hex code. Same in line 29

Reply:
true, ill change that